src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java

```diff
@@ -6,5 +6,7 @@
 
 public class GraphQlResolver {
     public static final String RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\ResolverInterface";
+    public static final String BATCH_RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\Resolver\\BatchResolverInterface";
+    public static final String BATCH_SERVICE_CONTRACT_RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\Resolver\\BatchServiceContractResolverInterface";
```

Comment:
I did it in my PR. Please fix the only issue with a line in Quick Fix. Thank you!

Reply:
Yes, But I have improved the quick fix with adding a select box to select one of the expected interfaces currenly quick fix is only added for `\Magento\Framework\GraphQl\Query\ResolverInterface`. 
Thanks

Reply:
Cool, great job

Reply:
Is this still in progress?